src/uwtools/resources/rocoto.jsonschema

```diff
@@ -1,70 +1,251 @@
+            },
+            "required": [
+              "items"
+            ]
+          }
+        },
+        "log": {
+          "type": "string",
+          "format": "uri"
+        },
+        "entities": {
+          "type": "object"
+        }
+      },
+      "required": [
+        "attrs",
+        "cycledefs",
+        "log"
+      ]
     },
-    "type": "object",
-    "properties": {
-        "workflow": {
-            "type": "object",
-            "properties": {
-                "attrs": {
-                    "type": "object",
-                    "properties": {
-                        "realtime": {
-                            "type": "boolean"
-                        },
-                        "scheduler": {
-                            "type": "string",
-                            "enum": [
-                                "sge",
-                                "lsf",
-                                "lsfcray",
-                                "ll",
-                                "torque",
-                                "moabtorque",
-                                "moab",
-                                "pbspro",
-                                "slurm",
-                                "cobalt"
-                            ]
-                        },
-                        "cyclethrottle": {
-                            "type": "integer",
-                            "minimum": 0
-                        },
-                        "taskthrottle": {
-                            "type": "integer",
-                            "minimum": 0
-                        }
-                    },
-                    "required": ["realtime", "scheduler"]
+    "dependency": {
+      "type": "object",
+      "patternProperties": {
+        "^(and|or|not|nand|nor|xor)(_.*)?$": {
+          "$ref": "#/$defs/BoolLoop"
+        },
+        "^datadep(_.*)?$": {
+          "type": "object",
+          "properties": {
+            "attrs": {
+              "type": "object",
+              "properties": {
+                "age": {
+                  "$ref": "#/$defs/TimePattern"
                 },
-                "cycledefs": {
-                    "type": "object",
-                    "properties": {
-                        "groupname": {
-                            "type": "array",
-                            "items": {
-                                "$ref": "#/$defs/dateTimePattern"
-                            }, 
-                            "required": ["ref"]
-                        },
-                        "required": ["items"]
-                    }
+                "minsize": {
+                  "type": "string"
+                }
+              },
+              "additionalProperties": false
+            },
+            "text": {
+              "type": "string"
+            }
+          },
+          "additionalProperties": false
+        }
+      },
+      "properties": {
+        "taskdep": {
```

Comment:
The `taskdep` entry is another one that can follow the `_arbitrary_string` suffix pattern.

Actually all of these below here can...

Reply:
oh wow, all of them? Ok, can do